frontend/src/scenes/experiments/experimentLogic.tsx

```diff
@@ -1331,13 +1363,22 @@ export const experimentLogic = kea<experimentLogicType>([
         getMetricType: [
```

Comment:
Might want to rename this to `getExperimentQueryType` in a follow-up.

Reply:
yep, noted it down